nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -241,34 +242,59 @@ where
                         db_creators.push(asset_creators::ActiveModel {
                             asset_id: Set(id_bytes.to_vec()),
                             creator: Set(c.address.to_bytes().to_vec()),
+                            position: Set(i as i16),
                             share: Set(c.share as i32),
+                            slot_updated: Set(Some(slot_i)),
+                            ..Default::default()
+                        });
+
+                        db_verified_creators.push(asset_creators::ActiveModel {
+                            asset_id: Set(id_bytes.to_vec()),
+                            creator: Set(c.address.to_bytes().to_vec()),
                             verified: Set(c.verified),
-                            seq: Set(seq as i64), // do we need this here @micheal-danenberg?
-                            slot_updated: Set(slot_i),
-                            position: Set(i as i16),
+                            seq: Set(Some(seq as i64)),
                             ..Default::default()
                         });
+
                         creators_set.insert(c.address);
                     }
 
                     let query = asset_creators::Entity::insert_many(db_creators)
                         .on_conflict(
                             OnConflict::columns([
                                 asset_creators::Column::AssetId,
-                                asset_creators::Column::Position,
+                                asset_creators::Column::Creator,
                             ])
                             .update_columns([
-                                asset_creators::Column::Creator,
+                                asset_creators::Column::Position,
                                 asset_creators::Column::Share,
+                                asset_creators::Column::SlotUpdated,
+                            ])
+                            .to_owned(),
+                        )
+                        .build(DbBackend::Postgres);
+                    txn.execute(query).await?;
+
+                    let mut query = asset_creators::Entity::insert_many(db_verified_creators)
```

Comment:
So we're re-inserting the same creators again? This code would benefit from some comments to explain this approach. It doesn't seem intuitive (to me at least).

Reply:
The second time we are just actually updated the `verified` and `seq` fields.  The reason the `ActiveModel` used in the second Vec still contains the `asset_id` and the `creator` is really just a matter of locating the record to update. I did it like this to keep it consistent with the rest of the code using the new ordering strategy, which is all using `upserts`. It is also because of the code structure that was there which was building up a Vec and then trying to do a single database call of `insert_many`.

If we think this pattern of `upserting` even when we know the record exists is too inefficient, I could change it around to do just do an `update` to set the `verified` status after the base information is `upserted`. But then I also have to refactor the existing looping structure a bit.